src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionWarnOthersViewController.swift

```diff
@@ -53,18 +53,26 @@ class ExposureSubmissionWarnOthersViewController: DynamicTableViewController, EN
 		exposureSubmissionService = exposureSubmissionService ?? (navigationController as? ExposureSubmissionNavigationController)?.exposureSubmissionService
 	}
 
-	// MARK: - ExposureSubmissionService Helpers.
```

Comment:
why do you remove this MARK?

Reply:
Ah, thanks for the comment. Accident on my side 👍